src/ui/StackedBarChart.js

```diff
@@ -1,444 +1,481 @@
- WGL.ui.StackedBarChart = function(m, div_id, x_label, filterId, params) {
```

Comment:
I see there are lots of formatting changes. I know the formatting needs some update but if a larger reformatting is required then let's do it in a separate PR as it makes it hard to follow any real changes there. Can you try to update this so just the real changs will be visible?